README.md

```diff
@@ -99,6 +99,28 @@ your local docker devstack by following these steps:
 Your devstack Studio should now be using the production studio-frontend files
 built by your local checkout.
 
+## Testing in studio-frontend
+
+### Where do test files go?
+If you are developing a component and you are adding new `js` or `jsx` files,
+the test files would go in the same location as the file. It makes it easier
+to track and test. For example if you are developing a component `AssertsSearch`
```

Comment:
Nit: I think you need a comma after "For example", i.e. "For example, if you are developing...".